cachi2/core/package_managers/bundler/parser.py

```diff
@@ -71,7 +71,8 @@ def purl(self) -> str:
     @cached_property
     def remote_location(self) -> str:
         """Return remote location to download this gem from."""
-        return f"{self.source}/gems/{self.name}-{self.version}.gem"
```

Comment:
**Commit message:**

I'm missing any context in the commit message, so I had to look at the code to see what's going on based on some suspicious changes in this commit (adding commentaries on trailing slash etc.) so please provide some description, it helps reviewers who don't hold the necessary context in their mind anymore to get up to speed more quickly.

Reply:
I'm imagining something along the lines:
_Access all gems using direct download URL instead of going through the hosted package index._